plugins/modules/ndo_l3out_routed_sub_interface.py

```diff
@@ -0,0 +1,547 @@
+    suboptions:
+      uuid:
+        description:
+        - The UUID of the port channel.
+        - This parameter or O(port_channel.reference) is required.
+        type: str
+      reference:
+        description:
+        - The reference details of the port channel.
+        - This parameter or O(port_channel.uuid) is required.
+        type: dict
+        aliases: [ ref ]
+        suboptions:
+          name:
+            description:
+            - The name of the port channel.
+            type: str
+            required: true
+          template:
+            description:
+            - The name of the template that contains the port channel.
+            - This parameter or O(port_channel.reference.template_id) is required.
+            type: str
+          template_id:
+            description:
+            - The ID of the template that contains the port channel.
+            - This parameter or O(port_channel.reference.template) is required.
+            type: str
+  interface_group_policy:
+    description:
+    - The name of the interface group policy.
+    type: str
+  ipv4_address:
+    description:
+    - The IPv4 address of the interface.
+    type: str
+  ipv6_address:
+    description:
+    - The IPv6 address of the interface.
+    type: str
+  ipv6_link_local_address:
+    description:
+    - The IPv6 link-local address of the interface.
+    type: str
+  ipv6_dad:
+    description:
+    - Whether to enable IPv6 Duplicate Address Detection (DAD).
+    - If this parameter is unspecified during create, NDO defaults to O(ipv6_dad=enabled).
+    type: str
+    choices: [ enabled, disabled ]
+  mac:
+    description:
+    - The MAC address of the interface.
+    type: str
+  mtu:
+    description:
+    - The Maximum Transmission Unit (MTU) of the interface.
+    - Use O(mtu=inherit) to inherit the value configured under the fabric L2 MTU settings.
+    - The value must be 1, or in the range 576 - 9216 or O(mtu=inherit).
+    type: str
+  target_dscp:
+    description:
+    - The target Differentiated Services Code Point (DSCP) of the interface.
+    - If this parameter is unspecified during create, NDO defaults to O(target_dscp=unspecified).
+    type: str
+    choices:
+    - af11
+    - af12
+    - af13
+    - af21
+    - af22
+    - af23
+    - af31
+    - af32
+    - af33
+    - af41
+    - af42
+    - af43
+    - cs0
+    - cs1
+    - cs2
+    - cs3
+    - cs4
+    - cs5
+    - cs6
+    - cs7
+    - expedited_forwarding
+    - unspecified
+    - voice_admit
+  encapsulation_type:
+    description:
+    - The encapsulation type of the interface.
+    type: str
+    aliases: [ encap_type ]
+    choices: [ vlan, vxlan ]
+  encapsulation_value:
+    description:
+    - The encapsulation value of the interface.
+    - If O(encapsulation_type=vlan), this is the VLAN ID which must be in the range 1 - 4094.
+    - If O(encapsulation_type=vxlan), this is the VXLAN Network Identifier (VNI) which must be in the range 5000 - 16777215.
```

Comment:
->
The option O(encapsulation_type=vxlan), specifies VXLAN Network Identifier (VNI) which must be in the range 5000 - 16777215.